pkg/util/winutil/pdhutil/pdhcounter.go

```diff
@@ -90,14 +93,17 @@ func (p *PdhCounterSet) Initialize(className string) error {
 
 // GetUnlocalizedCounter wraps the PdhAddEnglishCounter call that takes unlocalized counter names (as opposed to the other functions which use PdhAddCounter)
 func GetUnlocalizedCounter(className, counterName, instance string) (PdhSingleInstanceCounterSet, error) {
+	// refresh PDH object cache (refresh will only occur periodically)
+	CachedRefreshPdhObjectCache()
```

Comment:
The standard `PdhAddCounter` call takes localized names. If you want to pass an English counter name on a system that uses a different language then you need to use `PdhAddEnglishCounter`. See https://docs.microsoft.com/en-us/windows/win32/perfctrs/creating-a-query

I opened an issue to make these names more clear (WA-52)

Reply:
Oh, woops, I misread your question as "Why is this function called `GetUnlocalizedCounter`"

It is called here because this function (unfortunately) does not call `Initialize()`. I think fixing this code structure should be part of the issue I mentioned before (WA-52).